README.md

```diff
@@ -1,40 +1,41 @@
 # Welcome to the lab on enabling GitHub Pages!
 
-- [About the codebase](#about-the-codebase)
-- [Lab instructions](#lab-instructions)
-- [Grading criteria](#grading-criteria)
-- [A note about usage of GitHub Actions](#a-note-about-usage-of-github-actions)
-- [References](#references)
+This lab is an exercise to check your knowledge on enabling GitHub Pages. It is automatically graded via a workflow once you have completed the instructions.
 
+**Quick links:**
+- [About this lab](#about-this-lab)
+- [Instructions](#instructions)
+- [Seeing your result](#seeing-your-result)
+- [Troubleshooting](#troubleshooting)
+- [Useful resources](#useful-resources)
 
-## About the codebase
-The codebase contains a starter GitHub pages site inside the `docs/` directory, but you can use any content you'd like as long as the conditions specified in the [grading criteria](#grading-criteria) are met. 
+## About this lab
 
-A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and altering their contents will affect the repository's ability to assess your lab and give feedback.
+:warning: A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and **altering their contents will affect the repository's ability to assess your lab and give feedback.**
 
-## Lab instructions
+:information_source: This lab utilizes [GitHub Actions]( https://docs.github.com/en/actions) to grade the exercise, which means that it may take the grading workflow a few seconds and sometimes minutes to run.
 
-1. Create your own copy of this repository.
-   - The easiest way to copy this repository is to click on **Use this template**. See *[Creating a repository from a template]* if you need assistance.
-2. Complete the [grading criteria](#grading-criteria) in your newly created repository.
-3. Your lab will be graded automatically when the grading criteria is met, this may take up to a minute.
-   - To see the results of your lab, click on the **Actions** tab, select the **Grading workflow**, and select the most recent workflow run. See *[Viewing workflow run history]* if you need assistance. 
-4. If you're having trouble getting started, run the troubleshooter: click on the **Actions** tab, select the **Grading workflow**, click on **Run workflow**, select the appropriate branch (usually `main`), and click on the **Run workflow** button. See *[Running a workflow on GitHub]* if you need asssistance.
+## Instructions
 
-## Grading criteria
+1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) option. Make sure you set your repository visibility to **Public** when asked.
```

Comment:
I suggest calling the UI element by its name to help with clarity of the instructions. 
```suggestion
1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button. Make sure you set your repository visibility to **Public** when asked.
```